CHANGELOG.md

```diff
@@ -105,6 +105,8 @@ operations (e.g. `PREFIX-AsyncOperationEcsLogs`).
     to granules table
 
 ### Added
+- **CUMULUS-3385**
+  - added upload_db_executions to dump large scale postgres executions
```

Comment:
Thinking about it a bit more, maybe this doesn't make sense. They're separate use cases. Yours is for a larger-scale test in an environment other than local (though could be used locally) and the other is just really for unit tests. 

Keeping this comment here for kicks but I think it might be distinct enough to be moot.

Reply:
yea, I have fixed the changeling, but the intent of this scrip tis different